windows/eflow_deploy/eflow_deploy.yml

```diff
@@ -26,7 +26,7 @@
             skip_reason: "Not Supported"
           when:
             - guest_os_edition | lower not in ['pro', 'enterprise']
-            - guest_os_product_type | lower == 'client'
+            - guest_os_product_type == 'client'
```

Comment:
The include_tasks names can also be added.

Reply:
Added.